src/uu/uniq/src/uniq.rs

```diff
@@ -247,6 +247,23 @@ fn opt_parsed<T: FromStr>(opt_name: &str, matches: &ArgMatches) -> UResult<Optio
     })
 }
 
+fn obsolete_skip_field(matches: &ArgMatches) -> Option<usize> {
```

Comment:
Could you document this function? It'd be nice to both have a docstring and some comments inside the function to explain how it works.